docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,34 +2,78 @@
+NOTE: The default index glob patterns defined for {siem-soln} events are
 `endgame-*`, `auditbeat-*`, `winlogbeat-*`, `filebeat-*`, `packetbeat-*`,
-and `apm-*-transaction*`. You can change the default glob patterns in {kib} -> 
+and `apm-*-transaction*`. You can change the default glob patterns in {kib} ->
 Management -> Advanced Settings -> `siem:defaultIndex`.
 
 
 [float]
 [[siem-overview-ui]]
 === Overview
 
-The Overview page provides a high-level view into security events available
+The Overview page provides a high-level view of security events available
 for analysis, and can help surface problems with data ingestion.
 
-Notice the *Timeline* on the right of the {siem-ui}. It is always available for
-starting an investigation, whether in its collapsed or expanded state. Identify
-items you want to dig into, and drag and drop them to the Timeline. Those items
-are waiting for you when you are ready to start your investigation.
+All histograms and graphs on the overview page contain an **Inspect** button. Use inspect to receive details on count statistics, and to copy the JSON request and response set by the SIEM UI.
+
+
+[float]
+[[search-overview]]
+==== Search
+
+Search for individual SIEM items with the benefit of *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* in the SIEM search bar. A date/time filter set to `Last 24 hours` is enabled by default, but can be changed to any time range. If you want to filter your search results with other fields, select **Add Filter**, followed by the field from which to filter and the operator (such `is not` or `is between`) for your query.
+
+To save specific filters and queries, click the Save icon and then **Save current query**.
+
+[float]
+[[timelines-overview]]
+==== Timelines
+
+Select the collapsable *Timeline* button on the right of the {siem-ui} to start an investigation. Identify
+items you want from a specific event. See the <<timelines-ui,Timelines>> section for more information.
+
+The most recent timelines you've accessed appear in **Recent Timelines**.
+
+[float]
+[[signals-count]]
+==== Signals count
+
+The signals histogram displays a count of all the Signals detected by the {siem-app} within a set time frame. In the **Stack by** dropdown, select specific parameters for which to visualize the signal count. For example, stack by `user.name` for the graph to visualize the number of signals attributed to a specific user.
+
+To create signal detection rules, see <<rules-ui-create,Managing Signal Detection Rules>>.
+
+[float]
+[[external-alerts-count]]
+==== External alerts count
+
+The external alerts histogram displays a count within a certain time frame. You can stack by two parameters: `event-category` and `event.module`.
+
+
+[float]
+[[events-count]]
+==== Events count
+
+The events count histogram displays a count of all events within a set time frame. You can stack by `event.action`, `event.dataset`, and the `event.module`.
```

Comment:
Consider replacing the `and` in:

> You can stack by `event.action`, `event.dataset`, and the `event.module`.

with `or`, because the user may only select one option at a time.